shell/src/test/java/alluxio/cli/fsadmin/command/ConfigurationCommandTest.java

```diff
@@ -0,0 +1,103 @@
+      List<String> expectedOutput = Arrays.asList("Alluxio configuration information: ",
+          "Property                                 Value                                     "
+              + "         Source                        ",
+          "alluxio.master.port                      19998                                     "
+              + "         DEFAULT                       ",
+          "alluxio.master.web.port                  19999                                     "
+              + "         DEFAULT                       ",
+          "alluxio.master.hostname                  localhost                                 "
+              + "         SITE_PROPERTY (/alluxio/conf/alluxio-site.properties)",
+          "alluxio.underfs.address                  hdfs://localhost:9000                     "
+              + "         SITE_PROPERTY (/alluxio/conf/alluxio-site.properties)",
+          "alluxio.logger.type                      MASTER_LOGGER                             "
+              + "         SYSTEM_PROPERTY               ",
+          "alluxio.master.audit.logger.type         MASTER_AUDIT_LOGGER                       "
+              + "         SYSTEM_PROPERTY               ");
+      List<String> testOutput = Arrays.asList(output.split("\n"));
+      Assert.assertThat(testOutput,
+          IsIterableContainingInOrder.contains(expectedOutput.toArray()));
+    }
+  }
+
+  /**
+   * @return configuration info list to test
+   */
+  private List<ConfigProperty> prepareConfigList() {
+    List<ConfigProperty> configList = new ArrayList<>();
+
+    ConfigProperty firstConfigProperty = new ConfigProperty().setName("alluxio.master.port")
+        .setValue("19998").setSource("DEFAULT");
+    ConfigProperty secondConfigProperty = new ConfigProperty().setName("alluxio.master.web.port")
+        .setValue("19999").setSource("DEFAULT");
+    ConfigProperty thirdConfigProperty = new ConfigProperty()
+        .setName("alluxio.master.hostname").setValue("localhost")
+        .setSource("SITE_PROPERTY (/alluxio/conf/alluxio-site.properties)");
+    ConfigProperty fourthConfigProperty = new ConfigProperty().setName("alluxio.underfs.address")
+        .setValue("hdfs://localhost:9000")
+        .setSource("SITE_PROPERTY (/alluxio/conf/alluxio-site.properties)");
+    ConfigProperty fifthConfigProperty = new ConfigProperty().setName("alluxio.logger.type")
+        .setValue("MASTER_LOGGER").setSource("SYSTEM_PROPERTY");
+    ConfigProperty sixthConfigProperty = new ConfigProperty()
+        .setName("alluxio.master.audit.logger.type")
+        .setValue("MASTER_AUDIT_LOGGER").setSource("SYSTEM_PROPERTY");
+
+    configList.add(firstConfigProperty);
+    configList.add(secondConfigProperty);
+    configList.add(thirdConfigProperty);
+    configList.add(fourthConfigProperty);
+    configList.add(fifthConfigProperty);
+    configList.add(sixthConfigProperty);
+    return configList;
```

Comment:
for building a constant list like this, it's cleaner to do it with a single statement like 
```java
private List<ConfigProperty> prepareConfigList() {
  return Arrays.asList(
      new ConfigProperty().setName("alluxio.master.port").setValue("19998")
          .setSource("DEFAULT"),
      new ConfigProperty().setName("alluxio.master.web.port").setValue("19999")
          .setSource("DEFAULT");
      ...);

Reply:
Much better!!! Done, thanks!